ui_components/src/home/Home.css

```diff
@@ -962,27 +988,73 @@ body {
         animation-delay: -1.5s !important;
         animation-timing-function: ease-in-out;
     }
+    .home_lantern_3::after {
+        content: '';
+        box-shadow: 0 0 45px 3px #f9f8dd, 0 0 20px 2px #c78b7b;
+        position: absolute;
+        width: 10px;
+        height: 40px;
+        left: 5px;
+        top: 5px;
+        animation-delay: -1.5s !important;
+        animation: floatUpDown 5s infinite, Glow 4s infinite;
+        animation-timing-function: ease-in-out;
+    }
 .home_lantern_4 {
     position: absolute;
-    transform: translate(-225px, -250px) scaleX(-1);
+    transform: translate(-225px, -263px) scaleX(-1);
 }
+    .home_lantern_4::before {
+        position: absolute;
+        top: calc(50% - 20px);
+        left: calc(50% - 20px);
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        background-color: #fff;
+        box-shadow: 0 0 60px 30px #fff, 0 0 100px 60px #f0f, 0 0 140px 90px #0ff;
+    }
     .home_lantern_4 img {
         position: absolute;
         animation: floatUpDown 5s infinite;
         animation-delay: -3s !important;
         animation-timing-function: ease-in-out;
-        max-height: 90px;
+        max-height: 60px;
+    }
+    .home_lantern_4::after {
+        content: '';
+        box-shadow: 0 0 45px 3px #f9f8dd, 0 0 20px 2px #c78b7b;
+        position: absolute;
+        width: 10px;
+        height: 40px;
+        left: 5px;
+        top: 5px;
+        animation-delay: -3s !important;
+        animation: floatUpDown 5s infinite, Glow 4s infinite;
+        animation-timing-function: ease-in-out;
     }
 .home_lantern_5 {
     position: absolute;
-    transform: translate(225px, -250px);
+    transform: translate(205px, -255px);
 }
     .home_lantern_5 img {
         position: absolute;
         animation: floatUpDown 5s infinite;
         animation-delay: -1s !important;
         animation-timing-function: ease-in-out;
-        max-height: 60px;
+        max-height: 55px;
+    }
+    .home_lantern_5::after {
+        content: '';
+        box-shadow: 0 0 45px 3px #f9f8dd, 0 0 20px 2px #c78b7b;
+        position: absolute;
+        width: 10px;
+        height: 40px;
+        left: 5px;
+        top: 5px;
```

Comment:
This part is the same for each of these, could we pull it into a `.home_lantern::after` class?

Reply:
Resolved in commit.